### PR TITLE
All past messed up uploads, idk if there's any problem, but im just gonna make this pr. Also im adding cruel keypad just because hes impatient.

### PR DESCRIPTION
**Pull request by Limeboy0603**
1. nonverbal simon pdf fix and upload
2. all ~fucking~ czech translations (wire placement, morse a maze, colored switches, the screw, nonogram)
3. ultimate cipher condensed and interactive

btw, since this pr included pr 494/499 commits, i think we should shut down 499
contact me on discord if there are any problems

### CODE REVIEW

**Comment by Limeboy0603** [comment]
#504

**Comment by Limeboy0603** [comment]
Hes pretty impatient tbh

File: HTML/Nonogram translated (Čeština — Malované Křížovky) optimalizováno.html [hunk 1]
```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<title>Malovaná Křížovka — Keep Talking and Nobody Explodes Mod</title>
```
Comment (red031000) [inline HTML/Nonogram translated (Čeština — Malované Křížovky) optimalizováno.html]: please take the file name title from here instead

File: HTML/Nonogram translated (Čeština — Malované Křížovky).html [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<title>Malovaná Křížovka — Keep Talking and Nobody Explodes Mod</title>
```
Comment (red031000) [inline HTML/Nonogram translated (Čeština — Malované Křížovky).html]: again, please take the file name from here

File: HTML/Nonogram translated (Čeština — Malované Křížovky) optimalizováno.html (hunk 1 above)
Comment (Limeboy0603) [reply HTML/Nonogram translated (Čeština — Malované Křížovky) optimalizováno.html]: also changing pdf names, thx for reminder

File: HTML/Nonogram translated (Čeština — Malované Křížovka) optimalizováno.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<title>Malovaná Křížovka — Keep Talking and Nobody Explodes Mod</title>
```
Comment (red031000) [inline HTML/Nonogram translated (Čeština — Malované Křížovka) optimalizováno.html]: please take the title from here

File: HTML/Nonogram translated (Čeština — Malované Křížovka).html [hunk 4]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang='en'>
+	<head>
+		<title>Malovaná Křížovka — Keep Talking and Nobody Explodes Mod</title>
```
Comment (red031000) [inline HTML/Nonogram translated (Čeština — Malované Křížovka).html]: same with this title

File: HTML/Ultimate Cipher condensed interactive (Fang).html [hunk 5]
```diff
@@ -0,0 +1,1849 @@
+﻿<!DOCTYPE html>
+<html>
+
+<head>
+	<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+	<meta charset="utf-8">
+	<title>The Ultimate Cipher - Keep Talking and Nobody Explodes Module</title>
```
Comment (red031000) [inline HTML/Ultimate Cipher condensed interactive (Fang).html]: The dash here should be an em dash, not a hyphen

**Comment by Limeboy0603** [comment]
i might have screwed up when pulling origin

**Comment by red031000** [comment]
if you could fix it, that'd be great

**Comment by samfun123** [comment]
Merged in b7f1f8d10409ea5e8674da0c128a9196a52d9f64

**Comment by Limeboy0603** [comment]
ok im deleting my fork and reset it then

**Pull request closed without merging**